### ISSUE TITLE
pip install fails for Python 3.9 on Linux

### ISSUE THREAD

**Issue by mhoekveen** [IOHprofiler/IOHexperimenter#101]
A wheel file for Linux for Python 3.9 (cp39) is missing from PyPi.

It also rejects building all the source tarballs because the version in metadata differs from the PyPi specified one. With an error like:
```
Requested ioh (...)ioh-0.3.2.1.tar.gz (...) has different version in metadata: '0.2.9.9'
```

This causes pip to eventually fall back to the source file for 0.0.0.1 which then fails to build.

A workaround for Linux users is to use Python 3.8, or compile their own wheel. The latter runs into similar problems with version needing some manual adjustment, but the former works just fine.

**Comment by jacobdenobel** [comment]
Hi @mhoekveen, this should be fixed in v0.3.2.2. Could you please try reinstalling? Let us know if you still recieve issues.

**Comment by mhoekveen** [comment]
The issue is fixed. Package now successfully installs with python 3.9. Thanks!
